reproducer.yml

```diff
@@ -55,3 +55,10 @@
   roles:
     - role: ci_setup
     - role: reproducer
+
+  post_tasks:
+    - name: Allow traffic from OSP VMs to OSP API (needed for shiftstack)
+      ansible.builtin.command:
+        cmd: iptables -I LIBVIRT_FWI 1 -o ocpbm -j ACCEPT
+      become: true
```

Comment:
care to add the `become: true` and `when: ...` just under the `name:` key? It makes things slightly more readable when we scroll over a receipt :).

Reply:
Done